astropy/utils/tests/test_misc.py

```diff
@@ -32,7 +32,11 @@ def test_api_lookup():
     objurl = misc.find_api_page(misc, 'dev', False, timeout=3)
 
     assert strurl == objurl
-    assert strurl == 'http://devdocs.astropy.org/utils/index.html#module-astropy.utils.misc'
+    assert strurl == 'http://devdocs.astropy.org/utils/index.html#module-astropy.utils.misc'  # noqa
+
+    # Try a non-dev version
+    objurl = misc.find_api_page(misc, 'v3.2.1', False, timeout=3)
```

Comment:
FWIW, I added a test here to mimic usage by @cdeil, which tries to pull the doc from release.